File: examples/python/simple_math.py
def add(a, b):
    return a + b

def multiply(a, b):
    return a * b

def factorial(n):
    if n <= 1:
        return 1
    return n * factorial(n - 1)

def main():
    print("Simple Math Operations")
    
    x = 5
    y = 3
    
    sum_result = add(x, y)
    print("5 + 3 =", sum_result)
    
    product = multiply(x, y)  
    print("5 * 3 =", product)
    
    fact = factorial(5)
    print("5! =", fact)
    
    print("Numbers 1 to 5:")
    for i in range(1, 6):
        print(i, end=" ")
    print()

if __name__ == "__main__":
    main()

File: examples/go/simple_math.go
package main

import "fmt"

func add(a int, b int) int {
    return a + b
}

func multiply(a int, b int) int {
    return a * b
}

func factorial(n int) int {
    if n <= 1 {
        return 1
    }
    return n * factorial(n-1)
}

func main() {
    fmt.Println("Simple Math Operations")
    
    x := 5
    y := 3
    
    sum := add(x, y)
    fmt.Printf("%d + %d = %d\n", x, y, sum)
    
    product := multiply(x, y)
    fmt.Printf("%d * %d = %d\n", x, y, product)
    
    fact := factorial(5)
    fmt.Printf("5! = %d\n", fact)
    
    // Simple loop
    fmt.Println("Numbers 1 to 5:")
    for i := 1; i <= 5; i++ {
        fmt.Printf("%d ", i)
    }
    fmt.Println()
}